spec/grape/api_spec.rb

```diff
@@ -3869,4 +3869,18 @@ def before
       expect(grape_api.eql?(MyAPI))
     end
   end
+
+  describe 'const_missing' do
+    class Mounted < Grape::API
```

Comment:
This pollutes global scope. Try `Class.new(Grape::API)` like the example above? You can give it a name with `const_set`. Another way to do it is to remove the class in an `after` block.

Reply:
> Try Class.new(Grape::API)

Yeah, I tried that initially but for some reason it doesn't give the same result.

Does this look right to you?
```ruby
let(:mounted) { Class.new(Grape::API) }
# ...
before { subject.mount mounted => '/const' }
```